pylintrc

```diff
@@ -143,6 +143,7 @@ disable=print-statement,
         too-many-statements,
         attribute-defined-outside-init,
         redefined-builtin,
+        dangerous-default-value,
```

Comment:
I'd like to keep this warning on by default; in general it's correct. In `baserepo.py` we can do something like `#pylint: disable=dangerous-default-value` to silence just those warnings.

Reply:
Done. I agree.